### PR TITLE
test: Add e2e tests using APKAM keys

### PR DESCRIPTION
**Pull request by gkc**
**- What I did**
For e2e tests:
- During setup, do enrollments for client and daemon atSigns
- Deny / revoke any other e2e_all enrollments that might be hanging around from a previous run
- If daemons are at least version 5.3.0 (Note: the current branch is so) then start them with APKAM keys file
- If clients are at least version 5.3.0 then start them with APKAM keys file

**- How I did it**
- new script, apkam_setup.sh to handle the enrollments 
- modified start_daemons.sh and all of the places which run an sshnp or npt client so they use an APKAM keys file if they are at least version 5.3.0

**- How to verify it**
e2e tests pass

### CODE REVIEW

File: packages/dart/noports_core/lib/src/common/features.dart [hunk 1]
```diff
@@ -33,8 +33,4 @@ extension FeatureDescription on DaemonFeature {
         return 'support requests for specific device ports';
     }
   }
-
-  void talk() {
-    print('meow');
-  }
```
Comment (gkc) [inline packages/dart/noports_core/lib/src/common/features.dart:39]: Removed code which was copy-pasted here from an example, by mistake

File: packages/dart/noports_core/lib/src/sshnpd/sshnpd_impl.dart [hunk 2]
```diff
@@ -425,14 +425,6 @@ class SshnpdImpl implements Sshnpd {
...
 
     // Validate the request payload.
-    //
-    // If a 'direct' ssh is being requested, then
-    // only sessionId, host (of the rvd) and port (of the rvd) are required.
-    //
-    // If a reverse ssh is being requested, then we also require
-    // a username (to ssh back to the client), a privateKey (for that
-    // ssh) and a remoteForwardPort, to set up the ssh tunnel back to this
-    // device from the client side.
```
Comment (gkc) [inline packages/dart/noports_core/lib/src/sshnpd/sshnpd_impl.dart]: The above comments referred to the Sshnp requests; this method is to handle an Npt request

File: packages/dart/sshnoports/bin/activate_cli.dart [hunk 3]
```diff
@@ -1,13 +1,12 @@
 import 'dart:io';
 
-import 'package:at_onboarding_cli/src/activate_cli/activate_cli.dart'
-    as activate_cli;
+import 'package:at_onboarding_cli/src/cli/auth_cli.dart' as auth_cli;
 
 Future<void> main(List<String> args) async {
   try {
-    await activate_cli.main(args);
+    exit(await auth_cli.main(args));
```
Comment (gkc) [inline packages/dart/sshnoports/bin/activate_cli.dart:7]: Use the new auth_cli from the at_onboarding_cli package. It is backwards-compatible

**Review by cpswan — DISMISSED** [review]

**Comment by xavierchanth** [comment]
@gkc what is holding this PR back?

**Review by xavierchanth — DISMISSED** [review]

**Comment by gkc** [comment]
> @gkc what is holding this PR back?
> 
> 

@XavierChanth as per notes in the PR description : 
Nothing, now that the other PR has been approved, other than I need to a) merge it, b) publish new version of at_onboarding_cli, c) update the at_onboarding_cli dependency here, d) have it reviewed again and then e) merged. I'll do all of that in the morning

**Comment by gkc** [comment]
> > @gkc what is holding this PR back?
> 
> @XavierChanth as per notes in the PR description : Nothing, now that the other PR has been approved, other than I need to a) merge it, b) publish new version of at_onboarding_cli, c) update the at_onboarding_cli dependency here, d) have it reviewed again and then e) merged. I'll do all of that in the morning

Have done all of that.

**Review by murali-shris — APPROVED** [review]

**Pull request merged**